src/zenhub.py

```diff
@@ -110,91 +196,67 @@ def _update_issue_pipeline(self, pipeline, pos=None):
-
-    def update_issue(self, points=None, pipeline=None, pipeline_pos=None, to_epic=False):
-        """
-        Update the information of a ZenHub Issue.
-        :param int points: The number of points the issue should have.
-        :param str pipeline: The pipeline the issue should move to.
-        :param str or int pipeline_pos: The issue's position in the new pipeline. 'top', 'bottom', or 0-based index.
-                                        If unspecified, defaults to 'top'.
-        :param bool to_epic: Should the ticket be changed into an Epic?
-        """
-        logger.info(f'Beginning updating {self.issue} in repo {self.repo_name}')
-
-        if points:
-            self._update_issue_points(points)
-        if pipeline:
-            self._update_issue_pipeline(pipeline, pipeline_pos)
-        if to_epic:
-            self._update_issue_to_epic()
+                f'Error occured when updating issue to epic. Status Code: {response.status_code}. Reason: {response.reason}')
 
-        logger.info(f'Finished updating {self.issue} in repo {self.repo_name}')
+    def get_epic_children(self):
+        """Fill in the self.children field with all issues that belong to this epic. Self must be an epic."""
 
+        r = requests.get(f'{self.url}{self.repo_id}/epics/{self.github_key}', headers=self.headers).json()
+        return [i['issue_number']for i in r['issues']]
 
-class ZenHubIssue(Issue):
-
-    def __init__(self, key: str = None, repo_name: str = None, response: dict = None):
+    def change_epic_membership(self, add: str = None, remove: str = None):
         """
-        Create an Issue object from an issue key and repo name or from a portion of a ZenHub API response
-
-        :param key: If this and repo_name are specified, make an API call searching by this issue key
-        :param repo_name: If this and key are specified, make an API call searching in this repo
-        :param response: If specified, don't make a new API call but use this response from an earlier one
+        Add a given issue to or remove it from this epic in ZenHub. ZenHub issues can belong to multiple epics.
+        :param add: If specified, add the given issue as a child of self
+        :param remove: If specified, remove the given issue from self epic
         """
-        super().__init__()  # Initiate the parent class, Issue
-
-        self.url = get_access_params('zenhub')['options']['server']
-        self.token = get_access_params('zenhub')['api_token']
-        self.headers = {'X-Authentication-Token': self.token}
-        self.github_repo_name = repo_name
 
-        if key and repo_name:
-            self.github_key = key  # this identifier is used by zenhub and github
-            response = requests.get(f"{self.url}{get_repo_id(repo_name)}/issues/{key}",
-                                    headers=self.headers).json()
-
-        self.status = response['pipeline']['name']
+        # IMPORTANT the dictionary values here have to be ints
```

Comment:
Rather than write a comment here, I would assert it, so `assert isinstance(var_name, int)`. That way you have the information, but you also make sure it is so.

Reply:
Is it okay to just remove the comment and make sure each is an int by doing `int(var_name)` where they are used in the dictionary?